src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
```

Comment:
According to the checklist, you should use a for loop for creating several objects of the same class. This will make your code more scalable and maintainable, especially if the number of objects changes in the future.